src/server/polaris-server.ts

```diff
@@ -46,6 +43,10 @@ export class PolarisServer {
             logger: config.logger || getDefaultLoggerConfiguration(),
             applicationProperties: config.applicationProperties || { version: 'v1' },
             allowSubscription: config.allowSubscription || false,
+            shouldAddWarningsToExtensions:
```

Comment:
change to 
`shouldAddWarningsToExtensions: config.shouldAddWarningsToExtensions || true`

Reply:
it's not good because when it's false it converts to true becuse of the 'or'